diff --git configure/os/CONFIG_SITE.linux-x86_64.linux-x86_64 configure/os/CONFIG_SITE.linux-x86_64.linux-x86_64
index 42cb444..aca858a 100644
--- configure/os/CONFIG_SITE.linux-x86_64.linux-x86_64
+++ configure/os/CONFIG_SITE.linux-x86_64.linux-x86_64
@@ -47,3 +47,5 @@ MOTIF_INC=/usr/include
 #PYTHON_DIR=/usr/lib/python2.6
 #PYTHON_INCLUDE=/usr/include/python2.6
 
+X11_LIB+=/usr/lib64/
+MOTIF_LIB+=/usr/lib64/
